src/runtime/schema/settings.ts

```diff
@@ -33,42 +43,85 @@ export type SettingsInput = {
     /**
      * todo
      */
-    default?: ConnectionConfig | false
+    default?: ConnectionSettings | false
     // Extra undefined below is forced by it being above, forced via `?:`.
     // This is a TS limitation, cannot express void vs missing semantics,
     // being tracked here: https://github.com/microsoft/TypeScript/issues/13195
-    [typeName: string]: ConnectionConfig | undefined | false
+    [connectionTypeName: string]: ConnectionSettings | undefined | false
   }
   /**
    * Should a [GraphQL SDL file](https://www.prisma.io/blog/graphql-sdl-schema-definition-language-6755bcb9ce51) be generated when the app is built and to where?
    *
    * A relative path is interpreted as being relative to the project directory.
+   *
    * Intermediary folders are created automatically if they do not exist
    * already.
    *
-   * @default false
+   * @default "api.graphql"
    */
   generateGraphQLSDLFile?: false | string
   /**
-   * A glob pattern which will be used to find the files from which to extract the backing types used in the `rootTyping` option of `schema.(objectType|interfaceType|unionType|enumType)`
+   * A glob pattern which will be used to find the files from which to extract the backing types. Backing types are used in/selected in  the rootTyping option of these schema methods:
+   *
+   * ```
+   * schema.objectType
+   * schema.interfaceType
+   * schema.unionType
+   * schema.enumType
+   * ```
```

Comment:
I'm surprised by just how bad jsDoc readability in VSCode feels. I'm taking advantage of keeping things on their own line and not using inline code. Inline code styling really messes with the typography.